### DISCUSSION TITLE
Welcome to Project: SCP!

### DISCUSSION THREAD

**Discussion by Starstriker112** [Starstriker112/projectscp#1] [Announcements]
Welcome to Project: SCP, a currently WIP sandbox building prison architect-type game where as the player you must build and manage a containment facility and contain anomalous objects, items and anomalous humanoids that threaten the world and public normalcy. You play as the site director of a new site that is being built from the ground up. 

Some features we have planned for the game so far:
- A building system that allows for many building options to suit the individual player's needs and building style
- An AI system where tasks can be assigned and they complete them, but still have enough agency to move about on their own.
- management mechanics, that include personnel, water, power and sewage and containment as well as a host of other management stuff that may be defined later on.
- Ability to create and organize Mobile Task Force teams to go out into the world and secure new anomalies for the facility

more will be planned and talked about as we go further and further into development.
We encourage all those that are interested in keeping up with the project and wondering about what will be included in the game feature and mechanics wise, to please leave a comment and we will do our best to answer you as quick as we can.

As of right now, the project is currently in active development, sprite assets are already well under way and coding has also recently started. We hope you will love our game just as much as we will love making it and presenting it to you feature complete. Thank you for reading and we hope to talk to you guys in the near future.

**Comment by SwiftDust** [comment]
This description is cool!

**Reply by Razboinicul** [reply]
That is true!

**Comment by Tiguina** [comment]


**Reply by Razboinicul** [reply]
https://discord.com/invite/JtBBrJMA7r

**Reply by Razboinicul** [reply]
Are you happy now?
